includes/discovery/bgp-peers/dell-os10.inc.php

```diff
@@ -97,8 +99,8 @@
         }
 
         // clean up peers
-        if (dbFetchCell('SELECT count(*) FROM `vrfs` WHERE `device_id` = ?', [$device['device_id']]) == 0) {
```

Comment:
For cleanup, what I usually do is store ids of valid rows, then delete everything that doesn't have those ids.
The below code seems like a very long way to do that.